devicetypes/tonesto7/nest-thermostat.src/nest-thermostat.groovy

```diff
@@ -340,7 +344,7 @@ def deviceVerEvent() {
 	if (devVer()) {
     	def cur = parent?.latestTstatVer().ver.toString()
     	def ver = (cur != devVer().toString()) ? "Device Type:\nv${devVer()}(Latest: v${cur})" : "Device Type:\nv${devVer()}(Current)"
-    	sendEvent(name: 'devTypeVer', value: ver, displayed: false, isStateChange: true)
```

Comment:
Ok... That's one of those ST Features that is poorly documented.  So that makes sense